clustoapi/apps/attribute.py

```diff
@@ -177,6 +178,26 @@ def add_attr(name):
     if not obj:
         return util.dumps(msg, status)
 
+    try:
+        json_kwargs = request.json
+    except ValueError as ve:
+        return util.dumps('%s' % (ve,), 400)
+
+    if json_kwargs:
+        # Pull the serialized json tuple off of the params so that we can merge it back in gracefully.
+        for key in kwargs.keys():
```

Comment:
So tldr on this is that request.params give you both forms and querystring, and if you add content-type as json, the querystring turns into a form, so yea, gotta cover both cases and this was then sane-ish way to do it.